src/multi_processing/batch_size_calc.py

```diff
@@ -143,8 +143,8 @@ def swap_position_size(custom_args, target_num_records):
     """
     start_date = custom_args['start_date']
     start_date = datetime.strptime(start_date, '%Y%m%d')
-    end_date = datetime.today()
-    num_dates = ((end_date-start_date).days)+1
+    end_date = datetime.now(timezone.utc).date()
+    num_dates = (end_date-start_date).days + 1
```

Comment:
Please add spaces either side of the minus sign, otherwise it looks a bit like a single variable called end_date-start_date